moveit_core/collision_detection_fcl/src/collision_common.cpp

```diff
@@ -463,37 +468,148 @@ bool distanceCallback(fcl::CollisionObject* o1, fcl::CollisionObject* o2, void*
 
   if (always_allow_collision)
   {
-    min_dist = cdata->res_->distance;
-    return cdata->done_;
+    return false;
   }
 
-  fcl::DistanceResult dist_result;
-  dist_result.update(cdata->res_->distance, nullptr, nullptr, fcl::DistanceResult::NONE,
```

Comment:
You dropped this `update` call. Can you elaborate on this?

Reply:
This was a change you made as part of the commit "replace distanceCallback with new (detailed) function".

Reply:
According to git blame, yes, these changes are done by me. However, I only cleaned up your changes in order to facilitate the review. However, this update function call was already missing in your `distanceDetailedCallback()` function. Only with my cleanup changes, I was able to notice this change... Hence, the question is, why was it removed in `distanceDetailedCallback`?

Reply:
Oh, after the object is constructed the only variable that actually needed to be changed is the min_distance which used by fcl to only do detailed distance calculations if two objects are within this distance. The update call was unnecessary since the object was just constructed.